app/views/planning_applications/_assessment_dashboard.html.erb

```diff
@@ -29,6 +29,11 @@
 </div>
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds application">
+    <% unless @planning_application.determined? || @planning_application.returned? || @planning_application.withdrawn? %>
```

Comment:
I think we should have a method on PlanningApplication called `cancelable?` or similar for this.

In fact, further to that, I _think_ AASM provides us `@planning_application.may_return?`

Reply:
I need to investigate further why `.may_return?` and `.may_withdraw?` didn't work as expected. For the moment I've gone with   `true unless determined? || returned? || withdrawn?`